package.json

```diff
@@ -12,6 +12,7 @@
     "build": "babel src --out-dir dist --source-maps",
     "test": "babel-node jasmine.js",
     "cover": "NODE_ENV=test nyc --reporter=lcov jasmine JASMINE_CONFIG_PATH=jasmine.json",
+    "cover-windows": "SET NODE_ENV=test && nyc --reporter=lcov jasmine JASMINE_CONFIG_PATH=jasmine.json",
```

Comment:
I work under windows and `cover` script couldn't be run. SO I added a `cover-windows` script.